scripts/run_e2e_bare.sh

```diff
@@ -14,4 +14,4 @@ fi
 
 echo "${test_flags}"
 go test -c -tags=bare -mod=vendor -v -o e2e-bare github.com/operator-framework/operator-lifecycle-manager/test/e2e
-./e2e-bare  -test.v -test.timeout 20m ${test_flags} -kubeconfig=${KUBECONFIG:-minikube.kubeconfig} -namespace=$(cat e2e.namespace) -olmNamespace=operator-lifecycle-manager
+./e2e-bare  -test.v -test.timeout 20m ${test_flags} -kubeconfig=${KUBECONFIG:-minikube.kubeconfig} -namespace=$(cat e2e.namespace) -olmNamespace=operator-lifecycle-manager -dummyImage=hang:10
```

Comment:
Does a hang.Dockerfile need to be built here so the image exists for the baremetal run?

Reply:
It's built in `build_local.sh`